kivymd/uix/behaviors/__init__.py

```diff
@@ -18,5 +18,5 @@
 )
 from .focus_behavior import FocusBehavior
 from .magic_behavior import MagicBehavior
-from .ripplebehavior import CircularRippleBehavior, RectangularRippleBehavior
+from .ripple_behavior import CircularRippleBehavior, RectangularRippleBehavior
```

Comment:
Have you checked to see if this breaks the library and demo application dependencies?